src/test/java/school/lemon/changerequest/java/generics/pairUtils/pairUtils.java

```diff
@@ -0,0 +1,19 @@
+package school.lemon.changerequest.java.generics.pairUtils;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+/**
+ * Created by User on 03.01.2017.
+ */
+public class pairUtils {
```

Comment:
Class names should start from Uppercase letter.
Also, all tests should end with `Test` suffix